tests/test_processing.py

```diff
@@ -20,7 +20,24 @@
 COST_URL = f"{constants.API_HOST}/v2/processing/processes/{PROCESS_ID}/cost"
 EXECUTION_URL = f"{constants.API_HOST}/v2/processing/processes/{PROCESS_ID}/execution"
 TITLE = "title"
-ITEM_URL = "https://item-url"
+COLLECTION_ID = str(uuid.uuid4())
+ITEM_ID = uuid.uuid4()
+COLLECTION_URL = f"https://collection-url/v2/assets/stac/collections/{COLLECTION_ID}"
+ITEM_URL = "https://item-url/"
+COLLECTION = pystac.Collection.from_dict(
```

Comment:
Let's rather have `collection = mock.MagicMock()` in the only test which uses it